code/modules/mob/living/silicon/pai/pai_shell.dm

```diff
@@ -0,0 +1,92 @@
+
+/mob/living/silicon/pai/proc/fold_out(force = FALSE)
+	if(emitterhealth < 0)
+		src << "<span class='warning'>Your holochassis emitters are still too unstable! Please wait for automatic repair.</span>"
+		return FALSE
+
+	if(!canholo && !force)
+		src << "<span class='warning'>Your master or another force has disabled your holochassis emitters!</span>"
+		return FALSE
+
+	if(holoform)
+		. = fold_in(force)
+		return
+
+	if(emittersemicd)
+		src << "<span class='warning'>Error: Holochassis emitters recycling. Please try again later.</span>"
+		return FALSE
+
+	emittersemicd = TRUE
+	addtimer(src, "emittercool", emittercd)
+	canmove = TRUE
+	density = TRUE
+	if(istype(card.loc, /mob/living))
+		var/mob/living/L = card.loc
+		if(!L.unEquip(card))
+			src << "<span class='warning'>Error: Unable to expand to mobile form. Chassis is restrained by some device or person.</span>"
+			return FALSE
+		else if(istype(card.loc, /obj/item/device/pda))
```

Comment:
This will never be true because you just checked if card.loc was a /mob/living ?